launcher/sdw_updater_gui/UpdaterApp.py

```diff
@@ -158,6 +189,26 @@ def exit_launcher(self):
         sys.exit()
 
 
+def _is_netcheck_successful() -> bool:
```

Comment:
Logically the `_is_netcheck_successful` seems like a utility to me, and therefore best resides in `sdw_util/`. That's a nit, so don't change it just for cleanliness's sake, but given the logging problems mentioned in this review on L206, maybe moving into utils will make the logging config easier to bootstrap.

Reply:
I left this as-is because this is currently the only place we have a need for connectivity checking in the Updater code, and I have a vague future wish to disentangle some of the dependencies in UI classes (so eg Updater.py can import Util.py, but maybe some day in the distant future the UpdaterApp, more UI-focused, won't have dependencies outside of the `sdw_updater_gui` package).  However, I agree that some methods (this `netcheck`, and maybe `launch_securedrop_client`) could belong elsewhere.